1.0/learn/how-to-generate-code-for-protocol-buffers/index.html

```diff
@@ -211,7 +211,7 @@
                               <div class="section">
                                  <h1 id="how-to-generate-ballerina-code-for-protocol-buffer-definition">How to generate Ballerina code for Protocol Buffer Definition</h1>
 
-<p>The <code class="highlighter-rouge">Protocol Buffers</code> to Ballerina tool provides capabilities to generate Ballerina source code for the Protocol
+<p>The <code class="highlighter-rouge">Protocol Buffers</code> to Ballerina Tool provides capabilities to generate Ballerina source code for the Protocol
```

Comment:
In these contexts it's just another tool, right? I wonder if we should leave these as is and use "Ballerina Tool" only for the CLI tool that allows the `ballerina` commands?

Reply:
Fixed.